integrations/symfony-bundle.rst

```diff
@@ -0,0 +1,197 @@
+Symfony Bundle
```

Comment:
should we add a link to the HTTPlug tutorial in this chapter? saying that this is explaining how to use the symfony integration, but basic usage is in the tutorial?